minimax.cpp

```diff
@@ -0,0 +1,29 @@
+#include<iostream>
+
+using namespace std; 
+
+int minimax(int depth, int nodeIndex, bool isMax, int scores[], int height) {
+
+	if (depth == height) 
+		return scores[nodeIndex]; 
```

Comment:
Please make it look like C++ code, not C
```suggestion
	if (depth == height) {
		return scores[nodeIndex];
	}
```
Please apply this suggestion in the code below.